main.nf

```diff
@@ -266,16 +264,18 @@ include { MARK_DUPLICATES }       from './modules/local/mark_duplicates' addPara
 ================================================================================
 */
 
-include { BUILD_INDICES } from './modules/subworkflows/build_indices'
+include { BUILD_INDICES } from './modules/local/subworkflow/build_indices'
 
 /*
 ================================================================================
                         INCLUDE nf-core PIPELINE MODULES
 ================================================================================
 */
 
-include { FASTQC }  from './modules/nf-core/fastqc'
-include { MULTIQC } from './modules/nf-core/multiqc'
+include { TRIM_GALORE }     from './modules/nf-core/software/trim_galore.nf'
+include { MARK_DUPLICATES } from './modules/nf-core/software/mark_duplicates'
```

Comment:
```suggestion
include { PICARD_MARKDUPLICATES } from './modules/nf-core/software/picard_markduplicates'
```

Reply:
it's actually the gatk one, as picard is in gatk now